### PR TITLE
[TF-TRT] Add python test for CombinedNMS

### PR DESCRIPTION
**Pull request by trevor-m**
This test will fail during INT8 calibration until the FakeGPU registration for CombinedNMS in #28745 is merged.

### CODE REVIEW

**Comment by aaroey** [comment]
So should I review this after that PR is merged?

**Comment by trevor-m** [comment]
> So should I review this after that PR is merged?

Yes please, or it may help to use this test to validate that #28745 fixes the problem with calibrating CombinedNms.

**Comment by samikama** [comment]
@aaroey You can start reviewing it now. It will only compile successfully once #28745 is merged since it uses same nms kernel to do box selection.

**Comment by trevor-m** [comment]
I've confirmed that the changes to t`ensorflow/core/kernels/non_max_suppression_op.cc` in #28745 fixes the problem with CombinedNMS and allows this test to pass.

**Review by aaroey — DISMISSED** [review]

File: tensorflow/python/compiler/tensorrt/test/combined_nms_test.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+from tensorflow.python.compiler.tensorrt.test import tf_trt_integration_test_base as trt_test
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import image_ops_impl
+from tensorflow.python.platform import test
+
+
+class CombinedNmsTest(trt_test.TfTrtIntegrationTestBase):
```
Comment (aaroey) [inline tensorflow/python/compiler/tensorrt/test/combined_nms_test.py]: Please implement ShouldRunTest() and let it return False if the TRT version is <5.1

**Review by aaroey — APPROVED** [review]

**Pull request merged**